perllib/FixMyStreet/Cobrand/Bromley.pm

```diff
@@ -397,6 +397,19 @@ sub munge_report_new_contacts {
     $self->SUPER::munge_report_new_contacts($categories);
 }
 
+sub updates_disallowed {
+    my $self = shift;
+    my ($problem) = @_;
+
+    # Only open waste reports
+    if (my $contact = $problem->contact) {
+        my $waste = grep { $_ eq 'Waste' } @{$problem->contact->groups};
```

Comment:
This pattern has cropped up a few times, I wonder if we should have a `Contact::has_group` method for simplicity's sake?